nginx/config/vhosts/pages.conf

```diff
@@ -42,7 +42,7 @@ server {
   }
 
   location /designstandards {
-    proxy_pass http://127.0.0.1:8081/designstandards;
+    proxy_pass http://localhost:8081/designstandards;
```

Comment:
In this one instance, yes, because of the proxy tricks we're playing with https://playbook.cio.gov/designstandards/